src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -1317,7 +1317,8 @@ def idxmin():
 
     def infer_objects():
         """
-        Attempt to infer better dtypes for object columns.
+        Attempt to infer better dtypes for object columns. This is not currently supported
```

Comment:
I am little bit confused, there seems no not implemented error raised in base.py, which means it is supported, right?

Reply:
Before this PR, our overrides had `@dataframe_not_implemented` and `@series_not_implemented`, which automatically raise NotImplementedError.